src/main/java/io/github/townyadvanced/townyresources/controllers/TownResourceCollectionController.java

```diff
@@ -100,8 +102,7 @@ private static List<ItemStack> buildItemStackList(Player player, Set<Entry<Strin
 
             // mythicmobs integration
             if (TownyResources.getPlugin().isMythicMobsInstalled()) {
-                ItemManager mythicItemManager = TownyResources.getPlugin().getMythicItemManager();
-                ItemStack mythicItem = mythicItemManager.getItemStack(materialName);
+                ItemStack mythicItem = MythicBukkit.inst().getItemManager().getItemStack(materialName);
```

Comment:
```suggestion
                ItemStack mythicItem = TownyResources.getPlugin().getMythicItemManager().getItemStack(materialName);
```
Can probably do this.